src/atomicui/organisms/RouteBox/RouteBox.tsx

```diff
@@ -257,20 +257,34 @@ const RouteBox: FC<RouteBoxProps> = ({
 			const obj = getDestDept();
 
 			if (obj?.DeparturePosition && obj?.DestinationPosition) {
+				const isModePedestrianOrScooter = mode === TravelMode.PEDESTRIAN || mode === TravelMode.SCOOTER;
+				const uturnAvoidanceOption = isModePedestrianOrScooter ? {} : { UTurns: routeOptions.avoidUTurns };
+
 				const params: CalculateRoutesCommandInput = {
 					Origin: obj.DeparturePosition,
 					Destination: obj.DestinationPosition,
 					TravelMode: mode,
 					Avoid: {
 						TollRoads: routeOptions.avoidTolls,
-						Ferries: routeOptions.avoidFerries
+						Ferries: routeOptions.avoidFerries,
+						DirtRoads: routeOptions.avoidDirtRoads,
+						Tunnels: routeOptions.avoidTunnels,
+						...uturnAvoidanceOption
```

Comment:
Any reason why we aren't supporting other avoidance options? 

```
      "CarShuttleTrains": boolean,       
      "ControlledAccessHighways": boolean,       
      "DirtRoads": boolean,       
      "Ferries": boolean,       
      "SeasonalClosure": boolean,      
      "TollRoads": boolean,       
      "TollTransponders": boolean,       
      "TruckRoadTypes": [ "string" ],       
      "Tunnels": boolean,       
      "UTurns": boolean, 
```

https://docs.aws.amazon.com/location/latest/APIReference/API_CalculateRoutes.html